### PR TITLE
Add welcome message and fix links part 2

### PR DESCRIPTION
**Pull request by cansavvy**
Two parts to this PR: 

## Adding welcome message 

Adding Jeff's welcome message that he typed up in the word doc he shared. 

The main thing that needs to be decided is what the method of contact that should be recommended on this page. I put the google form but I can change that depending on what folks say. 

## Broken link fixes 
Related to the changes of #22 as well. 

Those changes made some of the internal links not work because we didn't carry the changes everywhere. This PR is meant to make the pages work again.

### CODE REVIEW

File: _pages/about.md [hunk 1]
```diff
@@ -13,38 +13,16 @@ header:
...
+- **Open** - we are strong advocates for open science and reproducible research and will develop our work in the open, share the resources we develop with open source licenses, and distribute them on platforms to make it easy to find and use
+- **Sustainable** - we are in this for the long haul. We know that building data resources takes a huge commitment of time and effort. So we want to make sure that these resources are developed in a way that makes it possible to maintain them without overwhelming our team or yours!
 
+If you are interested in learning more you can read about [our team](https://hutchdatascience.org/our_team/), learn a new data skill from [our training resources](https://hutchdatascience.org/training/), join our [community](https://hutchdatascience.org/support/), learn more about the [diverse data efforts at the Hutch](https://fredhutch.org/data), our read about the [latest from the DaSL](https://hutchdatascience.org/news/)
```
Comment (cansavvy) [inline _pages/about.md:28]:
```suggestion
If you are interested in learning more you can read about [our team](https://hutchdatascience.org/ourteam/), learn a new data skill from [our training resources](https://hutchdatascience.org/training/), join our [community](https://hutchdatascience.org/support/), learn more about the [diverse data efforts at the Hutch](https://fredhutch.org/data), our read about the [latest from the DaSL](https://hutchdatascience.org/news/)  
```

File: _pages/about.md [hunk 2]
```diff
@@ -13,38 +13,16 @@ header:
...
-
-Are you a Fred Hutch staff member looking for more training opportunities, documentation or just have a data science question in general?  Fill out our [interest form and tell us about it.](https://forms.gle/EpWTSWdfnVy6ToY98)
-
-
-
-## News and Events
-See our [news page](/news/) for updates, announcements and events from the Fred Hutch Data Science Lab.  Or join us and the Fred Hutch Data community [Slack](https://fhdata.slack.com).
-
-
-See our Twitter list for updates from the data science community at the Fred Hutch and beyond.
```
Comment (cansavvy) [inline _pages/about.md:42]: Let me know if we do want the tweets back and if so where.

Comment (vortexing) [reply _pages/about.md:42]: Can we just toss the tweets back at the very bottom of the page?

Comment (vortexing) [reply _pages/about.md:42]: Will commit it with these other edits.

**Review by vortexing — APPROVED** [review]
Looks good to me.  Will defer to Jeff.

**Pull request merged**